extension/wasm_src/Cargo.toml

```diff
@@ -3,23 +3,36 @@ name = "macro_railroad_ext"
 version = "0.1.0"
 authors = ["Lukas Lueg <lukas.lueg@gmail.com>"]
 build = "build.rs"
-edition = "2018"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
 
 [dependencies]
+wasm-bindgen = "0.2"
 railroad = "0.1"
 macro_railroad = "0.1"
-stdweb = "0.4"
-syn = { version = "1.0", default_features=false }
-rand = { version = "0.7", features=["stdweb"] }
 
-[lib]
-crate-type = ["cdylib"]
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.6", optional = true }
```

Comment:
This is optional in the manifest, but never used in the `Makefile`, so we never actually use this. I'm not too shy about code-size, as the wasm-blob is only downloaded once with the extension and basically free after that. We probably want this feature enabled all the time?!

Reply:
Ok, sounds good.